openedx/core/djangoapps/theming/helpers.py

```diff
@@ -17,3 +18,35 @@ def get_value(val_name, default=None, **kwargs):
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
     return microsite.get_value(val_name, default=default, **kwargs)
+
+
+def get_template_path(relative_path, **kwargs):
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    """
+    return microsite.get_template_path(relative_path, **kwargs)
+
+
+def is_theming_enabled():
```

Comment:
@ziafazal I changed method name according to the comment at https://openedx.atlassian.net/browse/WL-277?focusedCommentId=165143&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-165143